src/settings.py

```diff
@@ -1,3 +1,5 @@
+from django.utils.translations import ugettext_lazy as _
```

Comment:
this fails the test while of course ``_ = lambda s: s`` did not - what am I mussing @intellectronica